README.md

```diff
@@ -97,8 +97,12 @@ To run tests, run the following command:
 
```

Comment:
- [ ] OPTIONAL
Kindly ensure to remove the Microverse template logo, on _line 4_ of your ReadMe. Optionally, you can replace it with your own logo if you have one.